packages/datadog-plugin-graphql/src/utils.js

```diff
@@ -26,8 +26,15 @@ function extractErrorIntoSpanEvent (config, span, exc) {
 
   if (config.graphqlErrorExtensions) {
     for (const ext of config.graphqlErrorExtensions) {
+      const value = exc.extensions[ext]
+
       if (exc.extensions?.[ext]) {
```

Comment:
Why does the `exc.extensions?.[ext]` check on line 31 have a defensive `?.` but the one on 29 doesn't?
This must mean the case where `exc.extensions` doesn't exist must not be tested.

At any rate shouldn't line 31 look like this?

```js
if (value) {
```

Reply:
Good catch, I placed that line in the wrong place, fixing.